### PR TITLE
JIRA-OSDOCS2693: Updated the PrivateLink firewall ports for Splunk operator

### PR DESCRIPTION
**Pull request by sayjadha**
This PR is to address JIRA issue: https://issues.redhat.com/browse/OSDOCS-2693

Topic: https://deploy-preview-40643--osdocs.netlify.app/openshift-rosa/latest/rosa_getting_started/rosa-aws-prereqs.html#osd-aws-privatelink-firewall-prerequisites

Exact change: Section "AWS PrivateLink firewall prerequisites" > Procedure > Step 7 table > Rows 4, 5, and 6 > Updated the PrivateLink firewall ports for Splunk operator.

Repo: Request cherrypick to enterprise-4.9 and enterprise-4.10

### CODE REVIEW

File: modules/osd-aws-privatelink-firewall-prerequisites.adoc [hunk 1]
```diff
@@ -262,8 +262,15 @@ If you do not use a default Red Hat NTP server, verify the NTP server for your p
...
 |443
 
-|`.osdsecuritylogs.splunkcloud.com`
-`inputs1.osdsecuritylogs.splunkcloud.com`
+|`*.osdsecuritylogs.splunkcloud.com`
+|Required. Used by the `splunk-forwarder-operator` as a logging forwarding endpoint to be used by Red Hat SRE for log-based alerting.
+|443 and 9997
+
+|`http-inputs-osdsecuritylogs.splunkcloud.com`
+|Required. Used by the `splunk-forwarder-operator` as a logging forwarding endpoint to be used by Red Hat SRE for log-based alerting.
```
Comment (mburke5678) [inline modules/osd-aws-privatelink-firewall-prerequisites.adoc:270]: s/logging forwarding endpoint/log forwarding endpoint ?  Logging forwarding sounds odd to me. Might be right. I'm not sure.

**Comment by mburke5678** [comment]
@sayjadha One small nit. Otherwise LGTM

File: modules/osd-aws-privatelink-firewall-prerequisites.adoc (hunk 1 above)
Comment (sayjadha) [reply modules/osd-aws-privatelink-firewall-prerequisites.adoc]: Good catch, @mburke5678! I've updated the phrase to 'log forwarding endpoint'.

**Comment by sayjadha** [comment]
Thank you for the review, @mburke5678. Awaiting QE review, and will let you know when the PR is ready for merge. Request cherrypick to enterprise-4.9 and enterprise-4.10 branches.

**Comment by mrbarge** [comment]
Change looks fine to me, thanks for doing this!

**Comment by yuwang-RH** [comment]
LGTM.

**Comment by sagidlow** [comment]
LGTM

**Pull request merged**